Build/Makefile.Linux

```diff
@@ -1,7 +1,7 @@
 
 CC := clang
 AR := ar
-COMPILER := $(shell $(CC) --version | grep "Target:" | cut -d ' ' -f 2)
+COMPILER := $(shell $(CC) -dumpmachine)
```

Comment:
This works with clang so we can potentially update the Makefile.Darwin to use the same command.

Reply:
yeah, I just afraid to do this because can't test it. But what about to remove the `CC := clang` variable and use system's cc?

Reply:
`clang -dumpmachine` seems to be working for me. But let's leave clang as the default compiler for Darwin. Will do some more testing on my end and then merge the change. Thanks for this change.

Reply:
@stokito I got a chance to do more testing and this works as intended for both `clang` and `cc`. Could you please make the similar change for Makefile.Darwin and we can then merge this change?

Reply:
2 min

Reply:
ok, I made the same change for Darwin and Raspi and made a force push